ML App SwiftUI/ML App SwiftUI/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  ML App SwiftUI
+//
+//  Created by Jose Chavez Aparicio on 20/10/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
We don't need a UIVIewController for the SwiftUI side
Also, rename the folder, it is better to have no spaces between words, so just call it SwiftUI

Reply:
Controller deleted and created SwiftUIview
Folder name updated